pallet/src/tests/signer.rs

```diff
@@ -448,7 +533,8 @@ fn verify_old_multi_signer_requests_getting_removed() {
                 no_type_args!(),
                 &alice_addr_mv,
                 &dave_addr_mv,
-                &eve_addr_mv
+                &eve_addr_mv,
+                &2u8
```

Comment:
Is this supposed to be a lock ID?

`2u8` provides no context - let's follow it with a comment or either provide a context via the good variable name that stores it or via the macro or via just some random constant

Reply:
Done